fx_crash_sig/symbolicate.py

```diff
@@ -75,7 +74,7 @@ def __get_symbolication_request(self, stack_traces):
                 if 'module_index' not in src_frame:
                     continue
 
-                module_index = src_frame['module_index']
+                module_index = src_frame['module_index'] or 0
```

Comment:
For some reason `src_frame['module_index']` can be None. I couldn't figure out why and decided to push off thinking about that and then forgot.

Either it's possible for frames to not be associated with a module or it's a bug in minidump-analyzer. I should double-check this.

Reply:
Frames found with cfi show up like this:

```
src_frame: {'ip': '0x600aa', 'module_index': None, 'trust': 'cfi'}
```

So that's cool--we can't do anything with that in symbolication. I'll fix this code so the None case is handled right.